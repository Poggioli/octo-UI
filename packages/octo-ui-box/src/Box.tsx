import { StyledBox } from "./styles";

const Box = StyledBox;

export { StyledBox };
export { Box };

