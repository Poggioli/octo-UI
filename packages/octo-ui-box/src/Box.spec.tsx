import { render } from "@testing-library/react";
import { Box } from "./Box";

describe("<Box />", () => {
  it(`GIVEN a <Box />
      WHEN render is called
      THEN should render a <Box />`, () => {
    const rendered = render(<Box />);
    expect(rendered.container).toMatchSnapshot();
  });

  it(`GIVEN a <Box />
      WHEN toString is called
      THEN return a string`, () => {
    expect(Box.toString()).toBe(".octo-UI-c-lesPJm");
  });
});
