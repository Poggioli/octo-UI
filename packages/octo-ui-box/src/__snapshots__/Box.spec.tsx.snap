// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> GIVEN a <Box />
      WHEN render is called
      THEN should render a <Box /> 1`] = `
<div>
  <div
    class="octo-UI-c-lesPJm"
  />
</div>
`;
